nagios-plugins-storage.spec

```
@@ -48,6 +48,9 @@ rm -rf %{buildroot}
 %doc LICENSE README.md
 
 %changelog
+* Tue Jun 18 2024 Andrea Manzi <andrea.manzi@egi.eu> - 0.1.1-0
```

Shouldn't it be 0.1.1-1 as per the release ?
```suggestion
* Tue Jun 18 2024 Andrea Manzi <andrea.manzi@egi.eu> - 0.1.1-1
```